projects/valtimo/open-zaak/src/lib/open-zaak-extension/open-zaak-type-link-extension.component.ts

```diff
@@ -130,12 +124,13 @@ export class OpenZaakTypeLinkExtensionComponent {
   }
 
   openModal() {
+    var zaakTypeLink = this.zaakTypeLink$.getValue()
```

Comment:
pass zaakTypeLink as a parameter from the template instead of using it like this

Reply:
Changed it